benchmarks/StreamThroughput.cpp

```diff
@@ -152,69 +153,69 @@ class BM_Subscriber : public yarpl::flowable::Subscriber<Payload> {
   std::atomic<size_t> received_;
 };
 
-class BM_RsFixture : public benchmark::Fixture {
- public:
-  BM_RsFixture()
-      : host_(FLAGS_host),
-        port_(static_cast<uint16_t>(FLAGS_port)),
-        serverRs_(RSocket::createServer(std::make_unique<TcpConnectionAcceptor>(
-            TcpConnectionAcceptor::Options(port_)))) {
-    FLAGS_minloglevel = 100;
-    serverRs_->start([](const SetupParameters&) {
-      return std::make_shared<BM_RequestHandler>();
-    });
-  }
+std::unique_ptr<RSocketServer> makeServer(folly::SocketAddress address) {
+  TcpConnectionAcceptor::Options opts;
+  opts.address = std::move(address);
 
-  virtual ~BM_RsFixture() {}
+  auto acceptor = std::make_unique<TcpConnectionAcceptor>(std::move(opts));
 
-  void SetUp(const benchmark::State&) noexcept override {}
+  auto server = std::make_unique<RSocketServer>(std::move(acceptor));
 
-  void TearDown(const benchmark::State&) noexcept override {}
-
-  std::string host_;
-  uint16_t port_;
-  std::unique_ptr<RSocketServer> serverRs_;
-};
+  auto responder = std::make_shared<BM_RequestHandler>();
+  server->start([responder](const SetupParameters&) { return responder; });
 
-BENCHMARK_DEFINE_F(BM_RsFixture, BM_Stream_Throughput)
-(benchmark::State& state) {
-  folly::SocketAddress address;
-  address.setFromHostPort(host_, port_);
+  return server;
+}
 
-  auto s = yarpl::make_ref<BM_Subscriber>(state.range(0));
+std::shared_ptr<RSocketClient> makeClient(folly::SocketAddress address) {
```

Comment:
the `makeServer` and `makeClient` methods are very similar to the ones in `test/RSocketTests.h`.  Worth reusing the code ?